app/src/main/java/pl/droidevs/books/domain/Book.java

```diff
@@ -8,6 +8,9 @@
     private final String title;
     private final String author;
     private final Category category;
+    private String year;
```

Comment:
I'd expect year to be just number. Why it should be string? I'm not a big fan of using strings when there are more appropriate data types available. I also wonder why this is not final? Is there a business scenario that allow to change the release year?

Reply:
I just thought that conversions from string to int could be avoided in this way. However I can fix it if it does not comply with the rules of the project.  
Year and publisher cannot change, however it is possible that the user makes a mistake while saving a book and wants to fix it. Year and publisher is information that can be mistaken sometimes. But I could make them final, it's ok.

Reply:
Domain objects should not be affected by the way they are presented to the user. Common approach is to create separate models just for the view, as shown here: https://youtu.be/16wPp91kyuE?t=15m22s

Reply:
I have to admit this has confused me a lot.. I tried to do it in a simpler way but realised it wasnt simple either...are there other resourses on this matter you would suggest?

Reply:
I have to admit this has confused me a little bit. I have tried a lot to save the year as int but something always goes wrong. I watched the video resource and it seems that you have to write a lot of aditional code for this very little change. Is this really necessary when year is not used for any calcuation processes where an integer is required? 
Of course if its more important than I am able to judge i can keep trying, although I believe this needs more experience with android programming.

Reply:
What do you find difficult about it? If it's hard for you we can leave it as it is and I'll fix it later 😉 
Take a look at: http://jtechies.blogspot.com/2012/07/item-50-avoid-strings-where-other-types.html